### ISSUE TITLE
Fix accessibility issues in website

### ISSUE THREAD

**Issue by djhoese** [vispy/vispy#2138]
I recently learned about the [pa11y project](https://github.com/pa11y/pa11y) for identifying accessibility issues on websites. I installed it and ran it against the vispy website and got the following errors. It'd be nice if someone could look into addressing these. Some of this stuff may be specific to the pydata theme we are using and may not be something we can easily control. If it is part of the pydata theme then we should likely submit a pull request (or at least create an issue on their [github repository](https://github.com/pydata/pydata-sphinx-theme)). Note this is only run against the home (index) page. Running it on other pages shows similar issues, but many more and most of them repeats of the same issue that could likely be fixed by a single CSS change.

Any fixes for this stuff should be done in the `vispy-website` repository as a pull request. I should also warn that I am not a sphinx expert or a pydata sphinx theme expert or an accessibility expert, but I will help where I can.

```
$ pa11y https://vispy.org/

Welcome to Pa11y

 > Running Pa11y on URL https://vispy.org/

Results for URL: https://vispy.org/

 • Error: The html element should have a lang or xml:lang attribute which describes the language of the document.
   ├── WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
   ├── html
   └── <html><head> <meta charset="utf-8...</html>

 • Error: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
   ├── WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
   ├── html > body > div:nth-child(3) > div > div:nth-child(1) > form
   └── <form class="bd-search d-flex align-items-center" action="search.html" method="get"> <i class="icon fas fa-search...</form>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.44:1. Recommendation:  change text colour to #757778.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > div:nth-child(3) > div > div:nth-child(2) > div:nth-child(1) > div
   └── <div class="tocsection onthispage pt-5 pb-3"> <i class="fas fa-list"></i...</div>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation:  change background to #0072ed.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── #HomeCarousel > div > div:nth-child(1) > div > div > p:nth-child(3) > a
   └── <a class="btn btn-primary" href="gallery.html" role="button">Check it out</a>

4 Errors
```

**Comment by unnati914** [comment]
I can work on this

**Comment by djhoese** [comment]
Great! Let me know if I can help. Like I said, some of these may be an issue with the pydata sphinx theme. Overall, we want to get rid of the issues in the rendered output (the HTML), but that will require changes in the themes and markup (restructuredtext) that we use to build the site.

**Comment by unnati914** [comment]
As you mentioned, I have to make changes to the website but I have to submit PR here.. right?

**Comment by unnati914** [comment]
in this repo right?

**Comment by djhoese** [comment]
Yes and no. VisPy's repository (this one) is structured in a complex way. It uses a "git submodule" to include the vispy-website repository (https://github.com/vispy/vispy-website) into this one under the `doc` directory. So if you follow these instructions:

https://vispy.org/installation.html#via-github

you should have the vispy repository with a `doc` folder. You can treat that `doc` folder like a regular git repository (`git checkout` a different branch, `git remote` to add your fork of the `vispy-website` repository). If you need more detail on the git stuff let me know. You can also fine out more info about forks here: https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/about-forks

Once you have that doc folder you should be able to do:

```
make html
```

to generate the website output which will land in `_build/html/` I think. If you open `index.html` in your browser that will be your own copy of the website. You may need to skip generating the gallery to save yourself time. I'll have to look up how to do that though.

**Comment by djhoese** [comment]
I think you can do:

```
make html SPHINXOPTS="-D plot_gallery=0"
```

but I haven't tested it. I know this is a lot of information so let me know what I can clear up.

**Comment by djhoese** [comment]
FYI I'm updating the `doc` submodule in #2188 so all of this complex git submodule won't be needed once that is merged.

**Comment by djhoese** [comment]
#2188 has been merged now. Making changes to the documentation should be much easier now.

**Comment by unnati914** [comment]
tysm

**Comment by djhoese** [comment]
This just came up on my twitter: https://github.com/pydata/pydata-sphinx-theme/pull/294

**Comment by djhoese** [comment]
And their roadmap for known issues: https://github.com/pydata/pydata-sphinx-theme/blob/master/docs/a11y-roadmap.txt
